.cargo/config

```diff
@@ -8,6 +8,7 @@ rustflags = [
   "-Wclippy::semicolon_if_nothing_returned",
```

Comment:
```suggestion
   "-Wclippy::semicolon_if_nothing_returned",
```

Reply:
as ride long

Reply:
Updated nesting alongside new rule in a separate commit